### PR TITLE
add doc to create landing page

### PR DESCRIPTION
**Pull request by EvgeniaOrfeeva**
Draft version for landing page. Open for discussion

### CODE REVIEW

**Comment by MalteJ** [comment]
Please remove the `docs/.DS_Store` file from your commit :)

**Comment by afritzler** [comment]
Added `.DS_Store` to `.gitignore`

File: docs/648992860640615517gardener_on_metal_-_intro.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+CPU Architectures
+-----------------
+
+Gardener on Metal is tested on intel and AMD x86_64 processors. Due to its modular architecture Gardener on Metal should be easily portable to ARM (aarch64) or RISC-V processors. This is on our roadmap but we have not set a deadline, yet.
```
Comment (dmvolod) [inline docs/648992860640615517gardener_on_metal_-_intro.md:26]:
```suggestion
Gardener on Metal is tested on Intel and AMD x86_64 processors. Due to its modular architecture Gardener on Metal should be easily portable to ARM64 (AArch64) or RISC-V processors. This is on our roadmap but we have not set a deadline, yet.
```

**Comment by MalteJ** [comment]
> Added `.DS_Store` to `.gitignore`

you still need to remove the file from the commit.

**Review by MalteJ — APPROVED** [review]

**Comment by MalteJ** [comment]
LGTM

**Review by afritzler — CHANGES_REQUESTED** [review]
Please rename this file to `index.md` and overwrite the old one.

File: docs/648992860640615517gardener_on_metal_-_intro.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- Bare Metal Machines
+- Virtual Machines
+- Block Storage ([Ceph](https://docs.ceph.com/en/latest/) based)
+- Object Storage ([Ceph](https://docs.ceph.com/en/latest/) based, S3 compatible)
+- Virtual Networks (including true IPv4/IPv6 Dual-Stack support)
+- Load Balancers
+- NAT Gateways
+- ...
+
+> Gardener on Metal is not a new Open Stack.
```
Comment (afritzler) [inline docs/648992860640615517gardener_on_metal_-_intro.md:26]: Do we really want to have this as block quotes? Two consecutive block quotes are violating the Markdown style guide: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md028---blank-line-inside-blockquote

Comment (EvgeniaOrfeeva) [reply docs/648992860640615517gardener_on_metal_-_intro.md:26]: Nope, don`t need them at all

**Comment by MalteJ** [comment]
We probably don't need the bullet points in the Compute / Network / Storage sections. It's not really a list but just different paragraphs. Besides the information is not complete, what a bullet list may imply.

File: docs/index.md [hunk 3]
```diff
@@ -1,10 +1,79 @@
...
+- Bare Metal Machines
+- Virtual Machines
+- Block Storage ([Ceph](https://docs.ceph.com/en/latest/) based)
+- Object Storage ([Ceph](https://docs.ceph.com/en/latest/) based, S3 compatible)
+- Virtual Networks (including true IPv4/IPv6 Dual-Stack support)
+- Load Balancers
+- NAT Gateways
 
-Coz it's cool ...
+### Gardener on Metal is not a new Open Stack.
```
Comment (afritzler) [inline docs/index.md:25]:
```suggestion
### Gardener on Metal is not a new Open Stack
```

File: docs/index.md [hunk 4]
```diff
@@ -1,10 +1,79 @@
...
+The target of Gardener on Metal is not to virtualize or consolidate your existing on-prem infrastructure,
+but to be a slim deployment target for your cloud native workload. This allows us to use some short-cuts.
+For example onmetal's virtual networks have no support for Layer 2 Ethernet broadcasts. Instead the networks are
+Layer 3 only. This means all traffic between the instances is routed. The infrastructure therefore does not need to
+manage IP to MAC address resolutions, caching and cache invalidation. There is no need to care about multiplying
+packets of broadcasts, which could lead to micro-congestions and noisy-neighbor issues anymore.
 
+### Gardener on Metal does not support live migration of VMs.
```
Comment (afritzler) [inline docs/index.md:33]:
```suggestion
### Gardener on Metal does not support live migration of VMs
```

**Review by afritzler — CHANGES_REQUESTED** [review]
Trailing '.' in headers should not be used in Markdown

**Comment by afritzler** [comment]
Alright, all checks are passing and the content looks also good. I will squash merge this now.

**Review by afritzler — APPROVED** [review]

**Pull request merged**